State/Session.cs

```diff
@@ -217,6 +217,8 @@ public bool ReInitSessionWithNextBot(Account bot = null, double lat = 0, double
                                                                  $"XP : {XP:#,##0} ({(double)XP / ((double)NLevelXP) * 100:#0.00}%)\n" +
                                                                  $"SD : {SD:#,##0}", true).ConfigureAwait(false);
 
+            Thread.Sleep(1000); //Pauses execution for 1 sec. Personal use for TheWizard. I need a 1 sec gap between this and next logger line.
```

Comment:
Before Merge, if this is not needed, it's recommended to remove this.

Reply:
@TheWizard1328

Reply:
May not be needed but Is needed for another of my apps I am making that integrates Rockets logfile data into its database.